setup.py

```diff
@@ -7,16 +7,16 @@
     author="Saba Nafees",
     author_email="saba.nafees314@gmail.com",
     description="A PyPI package to compute multivariate tensor-based orthogonal polynomials for sequence data and map phenotypes onto sequence space.",
-    name="ortho_seq_code",
+    name="ortho_seqs",
     version="1.0.1",
     url="https://github.com/snafees/ortho_seqs",
     packages=find_packages(),
     keywords=[
         "regression",
-        "covariance",
+        "projections" "covariance",
```

Comment:
we are missing a comma and a newline here I believe

Reply:
yes, thank you, fixing it in another branch but will fix it here as well :)